### ISSUE TITLE
Server crash

### ISSUE THREAD

**Issue by dann1kid** [91loocekaj/vs-wildfarming#6]
1.2.5 Wild Farming

```
Game Version: v1.16.3 (Stable)
Loaded Mods: 
16.02.2022 9:25:17: Critical error occurred
System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в Vintagestory.API.Config.TranslationService.Load(Boolean lazyLoad)
   в Vintagestory.API.Config.Lang.Load(ILogger logger, IAssetManager assetManager, String defaultLanguage)
   в Vintagestory.Server.ServerSystemModHandler.OnLoadAssets()
   в Vintagestory.Server.ServerMain.EnterRunPhase(EnumServerRunPhase runPhase)
   в Vintagestory.Server.ServerMain.Launch()
   в ⁭‎​⁭⁪​‭‏⁯‮‪‭‏‮​‬⁮‍⁯‎⁫⁭⁭‭‬⁯​‬‏⁪‬⁫‍⁪‌​⁮‫‍⁭‮.⁬⁯⁪‌⁬⁯​‏‬⁮⁬⁫‌⁬⁮‏‭‏⁭‍‌‏‭‭⁬⁬⁬⁭‮‌‪‮⁮‌⁯⁮‌⁭⁫‮‮()
   в _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )```

**Comment by dann1kid** [comment]
i dont know why, but it maybe conflicts with another mod. Server starts after delete mod...
Here list of mods im using
```
External Origins in load order: modorigin@C:\GSM\servers\4\serverfiles\assets\creative\, modorigin@C:\GSM\servers\4\serverfiles\assets\survival\, mod@2xArmorDurability_v1.0.0.zip, mod@animalcages_v2.0.6.zip, mod@BackpackPackStandard_v1.0.3.zip, mod@BetterDrifters_v2.1.3.zip, mod@CaveContent_v1.0.0.zip, mod@CottageWindows_v1.0.1.zip, mod@CreaturesandCritters1.1.0.zip, mod@fancydoors_v1.16.0-1.1.0.zip, mod@MoreClasses_v1.2.4.zip, mod@MoreDrifterLoot-1.5.4.zip, mod@MoreRoads 1.4.3.zip, mod@necessaries_v1.16.0-1.1.1.zip, mod@qptech1.7.0.zip, mod@RangedWeapons-0.4.5.zip, mod@tradeomat_v1.16.0-1.6.0.zip, mod@wolffix_v1.0.0.zip, mod@alchemy-1.5.8.zip, mod@CarryCapacity-VS1.16.0-v0.6.4.zip, mod@FancyPlanters-v1.0.2.zip, mod@FixBadRecipes-v1.0.1.zip, mod@FixHandbookClutter-v1.0.1.zip, mod@Flint Chisel v1.0.2-VS1.16.0.zip, mod@FromGoldenCombs-1.16.0_v1.3.6-rc.3.zip, mod@medievalexpansion-3.7.1.zip, mod@MorePiles-v1.2.0.zip, mod@MorePilesQuasi-v1.0.1.zip, mod@petai_v1.3.4.zip, mod@PlaceEveryItem-v1.2.1.zip, mod@PlaceEveryItemFromMods-v1.0.1.zip, mod@PlaceEveryTool-v1.0.0.zip, mod@PlayerCorpse_v1.16.0-pre.8_v1.3.0.zip, mod@ResinRenaissance1.0.1-Fixed.zip, mod@RuinVariants-v1.0.0.zip, mod@StoneBrickRebalance-v1.0.0.zip, mod@StonePathVariants-v1.0.2.zip, mod@StoneQuarry_v1.16.3_v1.6.2.zip, mod@usefulstuff_v2.3.1.zip, mod@VanillaVariants-v1.1.2.zip, mod@wildfarming_v1.2.3 (2).zip, mod@WoodBarrels-v1.0.1.zip, mod@WoodChests-v1.0.2.zip, mod@workbench-expansion-1.4.1.zip, mod@xlib_v0.6.8.zip, mod@bricklayers-0.8.1.zip, mod@Bullseye 2.2.4.zip, mod@cats_v1.3.2_petai_v1.3.+.zip, mod@FeverstoneHorses-v1.2.5-1.16.x.zip, mod@Th3Essentials_1.1.5.zip, mod@wildcraft-v1.1.4.zip, mod@wolftaming_v1.3.1_petai_v1.3.x.zip, mod@xskills_v0.6.8.zip, mod@AlphaWeaponPack_v1.2.4.zip, mod@AnMiTech_v1.0.7.zip, mod@TemporalTinkerer_v2.3.4.zip
```

**Comment by 91loocekaj** [comment]
This has been fixed in 1.2.6. Please put bug reports on the forum page for this mod, I do not actively check Github.
